tests/ImageSharp.Tests/TestUtilities/TestEnvironment.cs

```diff
@@ -132,6 +135,95 @@ internal static string CreateOutputDirectory(string path, params string[] pathPa
             return path;
         }
 
+        /// <summary>
+        /// Creates Microsoft.DotNet.RemoteExecutor.exe.config for .NET framework,
+        /// When running in 32 bits, enforces 32 bit execution of Microsoft.DotNet.RemoteExecutor.exe
+        /// with the help of CorFlags.exe found in Windows SDK.
+        /// </summary>
+        internal static void PrepareRemoteExecutor()
+        {
+            if (!IsFramework)
+            {
+                return;
+            }
+
+            string remoteExecutorConfigPath =
+                Path.Combine(TestAssemblyFile.DirectoryName, "Microsoft.DotNet.RemoteExecutor.exe.config");
+
+            if (File.Exists(remoteExecutorConfigPath))
+            {
+                // already prepared
+                return;
+            }
+
+            string testProjectConfigPath = TestAssemblyFile.FullName + ".config";
+
+            File.Copy(testProjectConfigPath, remoteExecutorConfigPath);
+
+            if (Is64BitProcess)
+            {
+                return;
+            }
+
+            EnsureRemoteExecutorIs32Bit();
+        }
+
+        /// <summary>
+        /// Locate and run CorFlags.exe /32Bit+
+        /// https://docs.microsoft.com/en-us/dotnet/framework/tools/corflags-exe-corflags-conversion-tool
```

Comment:
Well this is clever...

Reply:
Could not find any simpler solution for now. BCL tests should also should run on 32bit Framework (in theory), but I'm not aware of such tricks in BCL test execution. Will dig deeper when I'll have time.